src/www/pages/components/disruption-indicators/index.njk.md

```diff
@@ -70,388 +79,469 @@
+
 <h3>When to use it?</h3>
-<ul>
-  <li>Showing a specific train, tram or road's disruption status</li>
-  <li>Example: homepage travel updates widget or major roadworks sidebar</li>
-</ul>
+
+- Showing a specific train, tram or road's disruption status
+- Example: homepage travel updates widget or major roadworks sidebar
+
 {# Research #}
+
 <h3>Research</h3>
-<ul>
-  <li>During user testing for the home page travel update widget, users preferred having a text description of the mode in addition to the modal icons, as the train and tram brand icons are very similar when small</li>
-</ul>
-<h3>Train</h3>
+
+- During user testing for the home page travel update widget, users preferred having a text description of the mode in addition to the modal icons, as the train and tram brand icons are very similar when small
+
+### Train
+
+{% endmarkdown %}
 {{
   compExample(
     [
       wmndsDisruptionIndicatorLarge({
         comment: '<!-- Train (good service) -->',
         mode: 'Train',
-        strongText: 'Good service',
-        text: 'Cross City Line',
+        primaryText: 'Good service',
+        contentText: 'Cross City Line',
         iconLeft: 'modes-isolated-rail',
         iconRight: 'general-success'
       }),
       wmndsDisruptionIndicatorLarge({
         comment: '<!-- Train (minor disruption) -->',
-        classModifier: 'warning',
+        classes: 'warning',
         mode: 'Train',
-        strongText: 'Minor disruption',
-        text: 'Cross City Line',
+        primaryText: 'Minor disruption',
+        contentText: 'Cross City Line',
         iconLeft: 'modes-isolated-rail',
         iconRight: 'general-warning-circle'
       }),
       wmndsDisruptionIndicatorLarge({
         comment: '<!-- Train (major disruption) -->',
-        classModifier: 'error',
+        classes: 'error',
         mode: 'Train',
-        strongText: 'Major disruption',
-        text: 'Cross City Line',
+        primaryText: 'Major disruption',
+        contentText: 'Cross City Line',
         iconLeft: 'modes-isolated-rail',
         iconRight: 'general-warning-triangle'
       }),
       wmndsDisruptionIndicatorLarge({
         comment: '<!-- Train (severe disruption) -->',
-        classModifier: 'severe',
+        classes: 'severe',
         mode: 'Train',
-        strongText: 'Severe disruption',
-        text: 'Cross City Line',
+        primaryText: 'Severe disruption',
+        contentText: 'Cross City Line',
         iconLeft: 'modes-isolated-rail',
         iconRight: 'general-warning-triangle'
       }),
       wmndsDisruptionIndicatorLarge({
         comment: '<!-- Train (delete) -->',
-        classModifier: 'error',
+        classes: 'error',
         mode: 'Train',
-        strongText: 'Major disruption',
-        text: 'Cross City Line',
+        primaryText: 'Major disruption',
+        contentText: 'Cross City Line',
         iconLeft: 'modes-isolated-rail',
         iconRight: 'general-trash',
         iconRightBtn: true
       })
```

Comment:
The classes for all of these are wrong as they are all showing as green: https://deploy-preview-568--wmnetwork.netlify.app/components/disruption-indicators/#train